src/hooks/matchmaking/start_matchmaking_find_sessions.cpp

```diff
@@ -28,6 +29,13 @@ namespace big
 					{
 						if (g_matchmaking_service->get_found_sessions()[i].is_valid)
 						{
+							auto host_rid =
+							    g_matchmaking_service->get_found_sessions()[i].info.m_net_player_data.m_gamer_handle.m_rockstar_id;
+
+							if (g.session_browser.without_host_modders && g_player_database_service->get_player_by_rockstar_id(host_rid)
+							    && g_player_database_service->get_player_by_rockstar_id(host_rid)->block_join)

```

Comment:
You're calling `get_player_by_rockstar_id`, this can be reduced to a single function call and saved into a variable before composing this statement.